depth_control_server:
  ros__parameters:
    tolerance: 0.02
    Kp: 0.4
    max_velocity: 0.25
    timeout: 20.0
    error_buffer: 0.02